nilearn/datasets/atlas.py

```diff
@@ -946,3 +946,52 @@ def fetch_atlas_surf_destrieux(data_dir=None, url=None,
 
     return Bunch(labels=annot_left[2],  map_left=annot_left[0],
                  map_right=annot_right[0], description=fdescr)
+
+
+def fetch_atlas_talairach(data_dir=None):
+    """Download the Talairach atlas.
+
+    Parameters
+    ----------
+    data_dir: string, optional (default=None)
+        Path of the data directory. Used to force data storage in a specified
+        location.
+
+    Returns
+    -------
+    sklearn.datasets.base.Bunch
+        Dictionary-like object, contains:
+        - maps: 3D Nifti image, values are integers corresponding to indices in
+          the list of labels.
+        - labels: list of tuples of strings. Each tuple contains five strings:
+          the hemisphere, the lobe, the gyrus, the tissue type and the Brodmann
+          area. The string '*' means 'undefined'. For example, the first index
+          in this list corresponds to the background and contains the tuple
+          ('*', '*', '*', '*', '*').
+        - description: a short description of the atlas and some references.
+
+    References
+    ----------
+    http://talairach.org/about.html#Labels
+
+    `Lancaster JL, Woldorff MG, Parsons LM, Liotti M, Freitas CS, Rainey L,
+    Kochunov PV, Nickerson D, Mikiten SA, Fox PT, "Automated Talairach Atlas
+    labels for functional brain mapping". Human Brain Mapping 10:120-131, 2000.`
+
+    `Lancaster JL, Rainey LH, Summerlin JL, Freitas CS, Fox PT, Evans AC, Toga
+    AW, Mazziotta JC. Automated labeling of the human brain: A preliminary
+    report on the development and evaluation of a forward-transform method. Hum
+    Brain Mapp 5, 238-242, 1997.`
+
+    """
+    atlas_url = 'http://www.talairach.org/talairach.nii'
+    data_dir = _get_dataset_dir(
+        'talairach_atlas', data_dir=data_dir)
+    atlas = _fetch_files(
+        data_dir, [('talairach.nii', atlas_url, {})])[0]
+    atlas = check_niimg(atlas)
+    labels = atlas.header.extensions[0].get_content()
```

Comment:
You should use get_header ```from nilearn._utils.compat import get_header```.
This way we could adapt to multiple versions of nibabel.